api/tests/prescription/learner-management/integration/domain/usecases/test-file/import-format-file/not-a-json.json

```diff
@@ -0,0 +1 @@
+this is not a json
```

Comment:
**THIS IS SPARTA** 

![image](https://github.com/user-attachments/assets/d8f06659-d535-46a8-acdf-17cfebb42c2c)
(pardon)